azerothcore/azerothcore-wotlk
# Eluna update

##### CHANGES PROPOSED:

-  Use the new config file read method.


###### ISSUES ADDRESSED: none


Closes : none


##### TESTS PERFORMED:  Successful build.Config can be loaded successfully on startup. You can also reload config with the command 'reload config'.







##### HOW TO TEST THE CHANGES: 

1. Need to use the latest'https://github.com/azerothcore/mod-eluna-lua-engine/commit/e541cb22b218d58a3cbe7c6a2f21893e12365363'
2. To build it.
3. View the output of 'worldserver'.
4. If you see the following prompt, it means that config is loaded normally.
```
Initialize Eluna Lua Engine...
[Eluna]: Searching scripts from `lua_scripts`
```
(I have finished testing in Win7)

##### KNOWN ISSUES AND TODO LIST:




##### Target branch(es):

Master
